checkov/common/checks/base_check_registry.py

```diff
@@ -196,3 +197,6 @@ def load_external_checks(self, directory: str) -> None:
                                 self.logger.error(f"Cannot load external check '{check_name}' from {check_full_path}", exc_info=True)
                             finally:
                                 BaseCheckRegistry.__loading_external_checks = False
+
+    def set_definitions_raw(self, definitions_raw: list[tuple[int, str]]) -> None:
```

Comment:
no need to create a method for this, just directly set the attribute, when needed